.readthedocs.yml

```diff
@@ -1,13 +1,32 @@
+# Docs:
+# https://docs.readthedocs.io/en/stable/config-file/v2.html
+# https://docs.readthedocs.io/en/stable/build-customization.html
+#
+# NOTE: Please, don't copy this file to build django docs on readthedocs.com
+# This is very specific to our needs (because this project is a template).
+# Also, we have a lot of configuration on rtd's admin side.
```

Comment:
Why then just to not move `docs` folder to root folder?

Reply:
Because we want `docs` to be present in the generated project.

Reply:
But why then you don't add `.readthedocs.yml` into the generated project?

Reply:
Because it is not needed. `.readthedocs.yml` is only needed for public (open-source) projects. But, 99.9% of our users generate private projects. And they just use `docs/` internally.